### PR TITLE
adding override to getskussmartlist to add an order by on sku code

### PR DESCRIPTION
**Pull request by jasperboyd**
This change is [Reviewable](https://reviewable.io/reviews/ten24/slatwall/4811)

### CODE REVIEW

File: model/entity/Product.cfc [hunk 1]
```diff
@@ -217,6 +217,13 @@ component displayname="Product" entityname="SlatwallProduct" table="SwProduct" p
 		return count;
 	}
 
+	public any function getSkusSmartList(){
```
Comment (rmarchandTen24) [inline model/entity/Product.cfc]: @sumitverma @gregmoser. Would this order by apply to 90% of the situations when calling product.getSkusSmartlist()?

Comment (sumitverma) [inline model/entity/Product.cfc]: I think it should be ok, since by default there was no sortorder.

Comment (gregmoser) [inline model/entity/Product.cfc]: Why don't we override this in the service?  That way anywhere that a sku smart list comes from will automatically be sorted by sku-code.

Comment (gregmoser) [inline model/entity/Product.cfc]: Just as an FYI, when this function isn't defined the smartList still comes from the skuService where it has been overwritten with an order defined.  Not sure if that matters to you @sumitverma

Comment (jasperboyd) [inline model/entity/Product.cfc]: @gregmoser that makes more sense I didn't realize they were both getting there smart list from the service, I'll update the pull request

**Pull request merged**